### ISSUE TITLE
Timeout waiting for boostrap-complete

### ISSUE THREAD

**Issue by bertinatto** [openshift/installer#819]
# Version

```console
$ openshift-install version
bin/openshift-install v0.5.0-master-40-gc0f55413e5eda382894f7a5a90f128c6ea4f9d8b
Terraform v0.11.8

Your version of Terraform is out of date! The latest version
is 0.11.10. You can update by downloading from www.terraform.io/downloads.html

$ ~/.terraform.d/plugins/terraform-provider-libvirt -version  # only needed if you're using libvirt
/home/fjb/.terraform.d/plugins/terraform-provider-libvirt was not built correctly
Compiled against library: libvirt 4.7.0
Using library: libvirt 4.7.0
Running hypervisor: QEMU 3.0.0
Running against daemon: 4.7.0

```

# Platform (aws|libvirt|openstack):

libvirtd

# What happened?

Started the installer and minutes later I got this message:

```
WARNING Failed to connect events watcher: Get https://test1-api.tt.testing:6443/api/v1/namespaces/kube-system/events?watch=true: dial tcp: lookup test1-api.tt.testing on [2001:730:3e92:1000::37]:53: no such host 
FATAL Error executing openshift-install: waiting for bootstrap-complete: timed out waiting for the condition 
```

I opened the `test1-bootstrap` VM window in `virt-manager` and saw this message:

![image](https://user-images.githubusercontent.com/1286651/49646134-827d1b00-fa1e-11e8-901d-d3d7b6914b08.png)

# Anything else we need to know?

This happened twice. I followed the guide closely, I don't think I missed anything.

**Comment by rishubhjain** [comment]
Have you done this step: https://github.com/openshift/installer/blob/master/docs/dev/libvirt-howto.md#set-up-networkmanager-dns-overlay ?

**Comment by rishubhjain** [comment]
@bertinatto Yes please try it.

**Comment by wking** [comment]
Fixed by #875, which drops "optional". Although we just need working DNS for the cluster's Kubernetes API, it doesn't matter how you accomplish that (dnsmasq, `/etc/hosts`, etc.).

See also #653.

/close
